_opcodes/op012-bg2.html

```diff
@@ -72,8 +72,33 @@
 <br/>1073741825  Disease Set to Value
 <br/>1073741826  Disease Set to Percentage
 <br/>1073741827  Disease Reduce by Percentage
-<br/>The Dice values can be used to inflict varying damage amounts - e.g. 2d6 would do 2-12 damage (Damage type is still specified by the 'Type' value).
-<br/>NB. 'Stunning' damage does not stun the target. Stunning damage cannot kill a creature, creatures are reduced to 1 HP and are knocked unconscious.
-<br/>NB. Using 'Magic Fire' or 'Magic Cold' as a damage type will result the game to crash if the damage kills the target.
-<br/>NB. If 'Damage Amount' and the Dice values are both non-zero, the total damage inflicted is: 'Damage Amount' + Dice value.
-<br/>NB. 'Damage Amount' can be positive or negative (though the damage sound will play in either case).
+
+{% capture note %}
+The Dice values can be used to inflict varying damage amounts - e.g. 2d6 would do 2-12 damage (Damage type is still specified by the 'Type' value).
+{% endcapture %}
+
+{% include info.html %}
+
+{% capture note %}
+'Stunning' damage does not stun the target. Stunning damage cannot kill a creature, creatures are reduced to 1 HP and are knocked unconscious.
+{% endcapture %}
+
+{% include note.html %}
+
+{% capture note %}
+Using 'Magic Fire' or 'Magic Cold' as a damage type will result the game to crash if the damage kills the target.
+{% endcapture %}
+
+{% include warning.html %}
+
+{% capture note %}
+If 'Damage Amount' and the Dice values are both non-zero, the total damage inflicted is: 'Damage Amount' + Dice value.
+{% endcapture %}
+
+{% include info.html %}
+
+{% capture note %}
+'Damage Amount' can be positive or negative (though the damage sound will play in either case).
+{% endcapture %}
+
+{% include info.html %}
```

Comment:
How does this look in the result? It seems it would be quite spammy. I'd just move the dice stuff into a paragraph, part of the description.